company/how-to-guides-for-staff/how-to-setup-a-1-1-channel.md

```diff
@@ -2,15 +2,15 @@
 description: 50% complete.
 ---
 
-# How to setup a 1-1 channel
+# How to set up a 1-1 channel
```

Comment:
Should this be in title case since it's the page title?

Reply:
Good catch!

Reply:
I just double-checked; all the "How to" guides are titled in sentence case so I've left it as-is.